lib/phlex/i18n.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Phlex
+	module I18n
```

Comment:
I’m not sure if I like `I18n` as a module name here. Perhaps `Translations` would be better.